workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in undere a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data that you port to Middleware with advanced filtering options by date range and metric value.
+
+# How Dashboards Work
+
+The Dashboard Builder is a Graphical User Interface (GUI) based approach that enables users to create complex query logic without the added stress of learning a new query language. The metrics made available to users when constructing dashboards are dynamically generated based on the data that users port to Middleware. 
```

Comment:
A good follow up sentence here would be, "For example, if you have insturmented Middleware's Host Agent and APM packages, you will have access to all of your host and application level data" and that gives us an opportunity to link to the host agent and apm. This little trick helps users find their way across documentation in case they land in a non-standard place, e.g. if their first doc is dashboarding.